css/main.scss

```diff
@@ -62,7 +62,8 @@ $grayscale-headline-font-family: $montserrat;
         "syntax-highlighting",
         "front_page",
         "post",
-        "summit";
+        "summit",
+        "summit-old";
```

Comment:
Instead of old, you could use 2017, since next year it won't scale :)

Reply:
Haha yeah, we'd have to change that every year; changed!